src/app/components/home/WalletPanel.tsx

```diff
@@ -162,12 +183,86 @@ const HeaderText = styled(Typography)`
   font-size: 14px;
   text-transform: uppercase;
   letter-spacing: 3px;
+
+  &:last-of-type {
+    padding-right: 25px;
+  }
 `;
 
 const DataText = styled(Typography)`
   font-size: 16px;
 `;
 
+export const notificationCSS = css`
```

Comment:
I'd like to move the `notificationCSS` in utils.ts.

Reply:
done